pennylane_lightning/lightning_qubit/lightning_qubit.py

```diff
@@ -226,10 +231,7 @@ def __init__(  # pylint: disable=too-many-arguments
 
             # Create the initial state. Internally, we store the
             # state as an array of dimension [2]*wires.
-            self._state = self._create_basis_state(0)
-            self._pre_rotated_state = self._state
-            self._c_dtype = c_dtype
-
+            self._qubit_state = _state_dtype(c_dtype)(self.num_wires)
```

Comment:
Do we need a test to silence the warning?

Reply:
This warning appeared because one of the upload code coverage steps failed. Coverage should be OK, as we can see in [e800725](https://github.com/PennyLaneAI/pennylane-lightning/pull/601/commits/e80072542253866431dcf9286834335666e20b4e).